packages/react/carbon__charts-react-tests.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { ScaleTypes } from "@carbon/charts/interfaces";
+import { LineChart } from "./dist";
+
+class App extends React.Component {
+  ref: React.RefObject<LineChart> = React.createRef();
+
+  state = {
+    data: {
+      labels: [''],
+      datasets: []
+    }
```

Comment:
this is the older data format that we deprecated many versions ago https://carbon-design-system.github.io/carbon-charts/?path=/story/tutorials--tabular-data-format

Reply:
are you able to update this to the tabular format?

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/867/commits/e9a53f4336a7af42a6b9caf555185df5624fc4de#diff-02ba3e7463b36e80fe42c20ff1e8d59c9575c7dbbd322bb6ce24630a2e240272